core/src/tile/tile.h

```diff
@@ -51,7 +49,7 @@ class Tile {
     float getScale() const { return m_scale; }
 
     /* Returns the reciprocal of <getScale()> */
-    float getInverseScale() const { return m_inverseScale; }
+    float getInverseScale() const { return 1.0f / m_scale; }
```

Comment:
There might be some cache locality taking into account.
Considering when we call this in https://github.com/tangrams/tangram-es/blob/rework-tile-states/core/src/style/polygonStyle.cpp#L94, there is already some reference to `_tile.getID()`, which would have brought in neighboring memory in cache/ram!

Anyhow, irrespective of the guess work of cache locality, what I am suggesting here are just best practices.